config/features.yml

```diff
@@ -697,6 +697,9 @@ features:
   mhv_transitional_medical_records_landing_page:
     actor_type: user
     description: Enables the transitional Medical Records page at /my-health/records
+  mhv_helpdesk_information_enabled:
+    actor_type: user
+    description: Enables the MHV helpdesk information
```

Comment:
```suggestion
    description: Enables the MHV helpdesk information on MHV-on-VA.gov Landing Page 
```

Reply:
Actually, I left it without that to leave it open for other apps to use it since the gate is the existence of the MHV helpdesk. So, instead of having a feature toggle for each app, we could have one for the MHV helpdesk.